### ISSUE TITLE
Training Evaluation when using pyTorch

### ISSUE THREAD

**Issue by marianturowski** [KIT-IAI/pyWATTS#32]
**What is your feature request related to?**

* Feature request

**Please describe the module/problem**

Based on feedback from @olineumann: Currently, one can only do random guessing on the number of epochs, batch size, ... because there is no feedback on how the training performs. Therefore, as a user of PyWATTS, it would be useful to be able to review the training process of pyTorch models, for example, in terms of a train/test loss curve.

**Describe the solution you'd like**

Maybe there is a way to add terminal printing? Maybe there is a way to plot train/test loss curves? Maybe there is a way to pass an individual method for evaluation after each epoch?

**Comment by benHeid** [comment]
Perhaps it can be a combined solution with #34 .

Perhaps, it would be possible to not only return the resulted dataarray but also additionally summary informations that are handled also by the step.
